onnx/onnx-ml.proto

```diff
@@ -547,7 +559,7 @@ message TensorProto {
   // float16 values must be bit-wise converted to an uint16_t prior
```

Comment:
Add `float8` to previous line? And add a similar line like this for `float8` ?

Reply:
done